### PR TITLE
feat:add installation for higress standalone in local docker environment

### LINKED ISSUE #572: hgctl 支持本地Docker 环境安装
## Why do you need it?
hgctl 支持本地docker 环境下安装

## How could it be?
1. 本地docker 环境下安装 local-docker  profile 如下：
 ```
# profile 名称
profile: local-docker
# 全局配置
global:
  # 安装模式 k8s/local-k8s/local-docker/local
  install: local-docker

# console 配置
console:
  # 本地安装时 console 端口
  port: 8080

# gateway配置
gateway:
  # 本地安装 Gateway 端口配置
  httpPort: 80
  httpsPort: 443
  metricsPort: 15020

# 控制台配置
controller:

# 本地安装存储配置
storage:
  # 后端存储类型，支持 nacos, 文件
  url: nacos://192.168.0.1:8848 #  file://opt/higress/conf
  # 后端存储类型为 naocs， 保存的命名空间
  ns: higress-system
  # 访问后端存储 用户名和密码
  username:
  password:
  # 后端存储加解密 key
  dataEncKey:

# higress & istio helm chart 配置
charts:
  higress:
    url: https://higress.io/helm-charts
    name: higress
    version: latest
  istio:
    url: https://istio-release.storage.googleapis.com/charts
    name: base
    version: 1.18.2
 ```

2. 集成 higress-standalone 安装

https://github.com/higress-group/higress-standalone/blob/main/bin/configure.sh

## Other related information
#229

### PR DESCRIPTION
**Pull request by 2456868764**
### Ⅰ. Describe what this PR did
1. support to install higress in local docker environment
2. add -f param to customize helm vlaues for installation and upgrade.
3. enable gatewayapi and reduce profile structure

### Ⅱ. Does this pull request fix one issue?

fixes #572 #570 #608

### CODE REVIEW

**Comment by johnlanni** [comment]
I think hgctl gateway-config/dashboard also need support local docker enviroment. Are you interested in implementing it, or creating an issue to track it?

**Comment by johnlanni** [comment]
cc @CH3CHO @Xunzhuo

**Comment by 2456868764** [comment]
> I think hgctl gateway-config/dashboard also need support local docker enviroment. Are you interested in implementing it, or creating an issue to track it?

i will create an issue to track and implement  it later.

**Review by Xunzhuo — APPROVED** [review]

File: pkg/cmd/hgctl/manifests/profiles/local-k8s.yaml [hunk 1]
```diff
@@ -1,37 +1,14 @@
...
   namespace: higress-system
-  istioNamespace: istio-system
 
 console:
   replicas: 1
-  serviceType: ClusterIP
-  domain: console.higress.io
-  tlsSecretName:
-  webLoginPrompt:
   adminPasswordValue: admin
```
Comment (johnlanni) [inline pkg/cmd/hgctl/manifests/profiles/local-k8s.yaml:11]: We should enable o11yEnabled when use local-k8s profile.

Comment (2456868764) [reply pkg/cmd/hgctl/manifests/profiles/local-k8s.yaml:11]: i add o11yEnabled property back in profile, and set true in local-k8s profile

**Review by johnlanni — APPROVED** [review]
Thanks!

**Pull request merged**